sdk/translation/Azure.AI.Translation.Text/src/Custom/TextTranslationClient.cs

```diff
@@ -118,6 +119,16 @@ public TextTranslationClient(TokenCredential credential, Uri endpoint, TextTrans
             }
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="TextTranslationClient"/> class.
+        /// </summary>
+        /// <param name="credential">DefaultAzureCredential</param>
+        /// <param name="endpoint">Service Endpoint</param>
+        /// <param name="options">Translate Client Options</param>
+        public TextTranslationClient(DefaultAzureCredential credential, Uri endpoint, TextTranslationClientOptions options = default) : this(credential as TokenCredential, endpoint, options)
```

Comment:
I'm not sure I understand the purpose of this code. What does this ctor allow the customer to do that they can't already do with the following ctor? `DefaultAzureCredential` is a `TokenCredential`.

```csharp
public TextTranslationClient(TokenCredential credential, Uri endpoint, TextTranslationClientOptions options = default) : this(endpoint, options)
```

Reply:
This won't pass architect review, for the reasons that Scott is calling out.   You'll want to remove this.

//cc: @tg-msft, @KrzysztofCwalina